src/app/App.tsx

```diff
@@ -45,13 +55,13 @@ const App: React.ComponentType = () => {
         }
     }, [ intl, location.pathname ]);
 
-    const hasReadPermissions = React.useMemo(() => {
+    const hasWritePermissions = React.useMemo(() => {
```

Comment:
I wonder if instead of changing it to write permission we should remove the read permissions from regular users.
This would be setting this to false: https://github.com/RedHatInsights/rbac-config/blob/master/configs/stage/roles/notifications.json#L22

We can do both stage and prod.

Reply:
Okay. Then I would just need to have `rbac ?` as the condition?

Reply:
ah ok yes, i see now.